src/components/personkort/PersonkortHeader/TildeltVeileder.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { Tag } from "@navikt/ds-react";
+import { useGetVeilederBrukerKnytning } from "@/components/personkort/hooks/useGetVeilederBrukerKnytning";
+
+const texts = {
+  tildeltVeileder: "Tildelt veileder:",
+  ufordeltBruker: "Ufordelt bruker",
+};
+
+export function TildeltVeileder() {
+  const veilederBrukerKnytning = useGetVeilederBrukerKnytning();
+
+  function tildeltVeilederText(): string {
+    if (!!veilederBrukerKnytning) {
+      return (
+        texts.tildeltVeileder +
+        " " +
+        veilederBrukerKnytning.tildeltVeilederident
```

Comment:
Det kommer kanskje visning av navn her etter hvert? (vi har egen query for å slå opp navn basert på ident)

Reply:
Hvis vi allerede har det i en hook så kan vi jo legge på navn med en gang 🙌